setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import find_packages, setup
+
+# Package meta-data.
+NAME = 'RePoE'
+DESCRIPTION = 'Repository of Path of Exile resources for tool developers'
+URL = 'https://github.com/brather1ng/RePoE'
+EMAIL = ''
+AUTHOR = 'brather1ng'
+REQUIRES_PYTHON = '>=3.6.0'
+VERSION = '1.0.0'
+
+# What packages are required for this module to be executed?
+REQUIRED = [
+   'PyPoe'
```

Comment:
```suggestion
   'PyPoE'
```
Its seems to be case insensitive, but can't hurt to have it correct